docs/commands/nop.md

```diff
@@ -3,21 +3,26 @@
 The `nop` command allows you to easily patch instructions with nops.
 
 ```
-nop [LOCATION] [--n NUM_ITEMS] [--b]
+nop [LOCATION] [--i ITEMS] [--f] [--n] [--b]
 ```
 
-`LOCATION` address/symbol to patch
+`LOCATION` address/symbol to patch (by default this command replaces whole instructions)
 
-`--n NUM_ITEMS` Instead of writing one instruction/nop, patch the specified number of
-instructions/nops (full instruction size by default)
+`--i ITEMS` number of items to insert (default 1)
+
+`--f` Force patch even when the selected settings could overwrite partial instructions
+
+`--n` Instead of replacing whole instructions, insert ITEMS nop instructions, no matter how many instructions it overwrites
+
+`--b` Instead of replacing whole instructions, fill ITEMS bytes with nops
 
-`--b` Instead of writing full instruction size, patch the specified number of nops
 
 ```bash
```

Comment:
This command has become a bit convoluted. Can you also add to the example what the commands do in plain english?

Reply:
me?? in english??? xDD oka, I will try

Reply:
done